xml/System.Formats.Cbor/CborConformanceMode.xml

```diff
@@ -135,15 +124,9 @@
           <para>Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.</para>
           <para>Extends lax conformance with the following requirements:</para>
           <list type="bullet">
-            <item>
-              <description>Maps (major type 5) must not contain duplicate keys.</description>
-            </item>
-            <item>
-              <description>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</description>
-            </item>
-            <item>
-              <description>UTF-8 string encodings must be valid.</description>
-            </item>
+            <item>Maps (major type 5) must not contain duplicate keys.</item>
```

Comment:
This change is not rendering correctly. It should be fixed in the source code.

![image](https://user-images.githubusercontent.com/24882762/141858500-539eea9d-90d8-4d93-aaf3-ed1c5ffac4e6.png)

Reply:
This looks better now - https://review.docs.microsoft.com/en-us/dotnet/api/system.formats.cbor.cborconformancemode?view=dotnet-plat-ext-5.0&branch=pr-en-us-7387.